src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -505,6 +515,64 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
       },
     },
   },
+  'Boolean Subtract': {
```

Comment:
Do we also want to add a warningMessage and make it unavailable in the command bar to non-dev and non-nightly users?